libpromises/match_scope.c

```diff
@@ -84,6 +84,15 @@ static int RegExMatchFullString(EvalContext *ctx, pcre *rx, const char *teststri
     }
 }
 
+/* Does not free rx! */
+bool PartialMatch(const pcre *rx, const char *teststring)
```

Comment:
What's this function for? Partial means not anchored? Add a small comment please.